nbs/core.ipynb

```diff
@@ -339,6 +339,8 @@
     "        self.id_col = id_col\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #111.            if static_df is None:
</pre><p>It may be easier to just use:</p><pre class="ql-syntax" spellcheck="false">if static_df is not None:
   # check cols of static_df
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>